### PR TITLE
(PDB-2353) support upgrades from 2.3.2

### PR DESCRIPTION
**Pull request by wkalt**
This reverts some of the combined migration to allow upgrades from 2.3.2 and
later.

### CODE REVIEW

**Comment by senior** [comment]
It looks like there were no database migrations from 2.3.2 through 2.3.7, is that why we're supporting back to 2.3.2?

**Comment by wkalt** [comment]
@senior the earliest PDB in PE 3.8.z is 2.3.2, so 2.3.2 is the actual requirement because we need to support upgrades from all z's according to the email. You're right that it doesn't make a practical difference though.

File: test/puppetlabs/puppetdb/scf/migrate_test.clj [hunk 1]
```diff
@@ -358,27 +452,6 @@
                   (map #(dissoc % :same))
                   set))))))
 
-(deftest test-migrate-from-unsupported-version
```
Comment (ajroetker) [inline test/puppetlabs/puppetdb/scf/migrate_test.clj]: Should we still have this test for old 1.x migrations?

**Comment by ajroetker** [comment]
Works for me (database upgrades fine) just the one comment about the test and then I think we're good to merge

**Pull request merged**